pkg/utils/logs/logs_test.go

```diff
@@ -0,0 +1,20 @@
+package logs
+
+import "testing"
+
+func TestLogContent(t *testing.T) {
+	logPrefix := "hotpot"
+	pid := "11111"
+	logLevel := "ERROR"
+	filename := "test.go"
+	line := "20"
+	funcname := "testFunc"
+	logMsg := "Hello, World"
+	error := "404"
+	logContent := GetLogContent(logPrefix, pid, logLevel, filename, line, funcname, logMsg, error)
```

Comment:
Why do we need to manually give pid, filename, line, funcname etc...? Please avoid that from the framework call. It can be automatically taken by the framework right? Please check how to optimize the same.
Why don't we use some standard go log packages and update as needed? (log, logrus, glog ..?)